package.json

```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-import": "2.7.0",
     "fs-extra": "4.0.2",
     "mocha": "3.5.3",
+    "webdriver-manager": "^12.0.6",
```

Comment:
This is unrelated and I don't understand why its required? That too on the base

Reply:
The build was failing because of the missing selenium lib, which I thing it's had just been fix by @pascalgrimaud  in #6474. On my dev machine I have the webdriver-manager installed globally and the test was passing. In package.json the webdriver-manager was missing and I was hopping to fix the build, which was obviously the wrong approach. I will revert the last commit immediately. The test are gone still not pass until the selenium bug is fixed. Should I wait until the selenium bug is fixed and then rebase on that or what will be the better approach. 10x

Reply:
You can revert this as @pascalgrimaud solution is the right one for this